pkg/osquery/table/slack_config.go

```diff
@@ -0,0 +1,122 @@
+package table
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"path/filepath"
+	"runtime"
+	"strconv"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	osquery "github.com/kolide/osquery-go"
+	"github.com/kolide/osquery-go/plugin/table"
+	"github.com/pkg/errors"
+)
+
+var slackConfigDirs = map[string][]string{
+	"windows": []string{"AppData/Roaming/Slack/storage"},
+	"darwin":  []string{"/Library/Application Support/Slack/storage"},
```

Comment:
```suggestion
	"darwin":  []string{"Library/Application Support/Slack/storage"},
```
Why the leading slash